src/types/timezone.jl

```diff
@@ -65,14 +67,8 @@ US/Pacific (UTC-8/UTC-7)
 TimeZone(::AbstractString, ::Class)
 
 function TimeZone(str::AbstractString, mask::Class=Class(:DEFAULT))
-    # Note: If the class `mask` does not match the time zone we'll still load the
-    # information into the cache to ensure the result is consistent.
-    tz, class = get!(_tz_cache(), str) do
-        tz_path = joinpath(_COMPILED_DIR[], split(str, "/")...)
-
-        if isfile(tz_path)
-            open(TZJFile.read, tz_path, "r")(str)
-        elseif occursin(FIXED_TIME_ZONE_REGEX, str)
+    tz, class = get(_TZ_CACHE, str) do
```

Comment:
I'm confused: why is there still a get() do here?

Shouldn't all of the timezones be pre-fetched now? Or do we only cache the variable time zones, not the FixedTimeZones?

Reply:
We use `get` here as we previously supported `TimeZone("UTC+4")` which isn't a time zone which would be automatically added to the cache. This allows this behaviour to continue but with the caveat that the construction of this new `FixedTimeZone` would have previously have been added to the cache but no longer. I very much suspect this is a non-issue as the caching would only be beneficial when calling `TimeZone("UTC+4")` multiple times.